### ISSUE TITLE
[suggestion]Superconductor recipes

### ISSUE THREAD

**Issue by Cinobi** [GTNewHorizons/NewHorizons#2475]
#### Which modpack version are you using?
Delta one
#
#### If in multiplayer; On which server does this happen? 
Delta
#
#### What did you try to do, and what did you expect to happen?
I think current recipe is not cool enough, it have some room for improvement.

Less logical way:
Lv pump - 3 output (present right now)
Mv pump - 6 output
Hv pump - 9 output
... 
More logical way:

Lv pump +3x1 wires - 3 output
Mv pump with 3x2 wires - 6 output
HV pump with 3x4 wires - 12 output
EV pump with 3x8 wires  - 24 output
IV pump with 3x12 wires - 36 output
...
![2018-01-21_03 14 17](https://user-images.githubusercontent.com/25028786/35189334-32c0b0b6-fe59-11e7-96db-09cc9accd41f.png)

**Comment by bartimaeusnek** [comment]
similar to this: https://github.com/GTNewHorizons/NewHorizons/issues/2433

**Comment by Dream-Master** [comment]
Yes but making recipes like this for all super conductors looks cooler

**Comment by Dream-Master** [comment]
We let it open until I add it.

**Comment by benstuwie** [comment]
id suggest changing the numbers a little. Once you're making superconductors in bulk, an mv pump isn't 2x as costly as an LV one, but an IV one is probably more than 1.5x an ev one

**Comment by Dream-Master** [comment]
@benstuwie Ok write down a list how you want to tier it like cinobi did.

**Comment by benstuwie** [comment]
Ok sorry probably shoulda had that with the first post. I've been crafting a lot of them recently for different things (using them for all my cables, and tons of UV parts/circuits) and a 12x modifier does seem like a huge increase. It's a buff on it's own just being able to not use tin for the rotors, lol. I'd say LV 2, MV3, HV 5, EV 6, IV 8. Titanium is so rarely used that at least I have tons of it, so EV parts are really easy to make. I'm pretty sure IV parts use the new rubbers, and graphene which is pain to make, so I'd give it a pretty significant bonus. Apart from the tin right now, they aren't that bad to craft in large quantities. I make them in batches of 4096 at a time, because any more will crash the server when loading in ae. Those take maybe 30min, which the longest part being the rings, because I don't have a ring processing array.

**Comment by bartimaeusnek** [comment]
maybe it should use the highest Superconductor mentioned here: #2433 instead of the wires?

**Comment by Dream-Master** [comment]
I like to see on all superconductors till mv/hv pumps in the recipes. Exception for redalloy and Redstonealloy wires

**Comment by repo-alt** [comment]
Please don't forget, if you are making existing superconductors "harder", they should still be craftable at LuV level, because they are used in fusion coils.

**Comment by Dream-Master** [comment]
Ok will change the recipe from the coils.
